wkcuber/convert_nifti.py

```diff
@@ -54,13 +60,41 @@ def create_parser():
         default=None,
     )
 
+    parser.add_argument(
+        "--write_tiff",
+        help="Output tiff dataset instead of wkw.",
+        default=False,
+        action="store_true",
+    )
+
+    parser.add_argument(
+        "--use_orientation_header",
+        help="Use orientation information from header to interpret the input data (should be tried if output orientation seems to be wrong).",
+        default=False,
```

Comment:
I set this to False for now, since other nifti files did not seem to use this header correctly If the output is flipped, one can try to toggle this flag.